source/_patterns/00-atoms/04-colors/02-brand-colors.mustache

```diff
@@ -0,0 +1,63 @@
+<hr>
+<h4>Colors - used Brand Colors</h4>
+<hr>
+
+<div class="mb-2 col-3" style="float:left;">
+    <div class="p-3 bg-orange text-light">Orange (Primary)</div>
+    <div class="p-3 bg-orange-80 text-light">80%</div>
+    <div class="p-3 bg-orange-60">60%</div>
+    <div class="p-3 bg-orange-40">40%</div>
+    <div class="p-3 bg-orange-20">20%</div>
+</div>
+
+<div class="mb-2 col-3" style="float:left;">
+    <div class="p-3 bg-dark-gray text-light">Dark Gray (Secondary)</div>
+    <div class="p-3 bg-dark-gray-80 text-light">80%</div>
+    <div class="p-3 bg-dark-gray-60 text-light">60%</div>
+    <div class="p-3 bg-dark-gray-40">40%</div>
+    <div class="p-3 bg-dark-gray-20">20%</div>
+</div>
+
+<div class="mb-2 col-3" style="float:left;">
+    <div class="p-3 bg-light-gray">Light Gray</div>
+    <div class="p-3 bg-light-gray-80">80%</div>
+    <div class="p-3 bg-light-gray-60">60%</div>
+    <div class="p-3 bg-light-gray-40">40%</div>
+    <div class="p-3 bg-light-gray-20">20%</div>
+</div>
+
+<div class="mb-2 col-3" style="float:left;">
+    <div class="p-3 bg-white">White</div>
+    <div class="p-3 bg-black text-light">Black</div>
+    <div class="p-3 bg-purple text-light">Purple</div>
+    <div class="p-3 bg-dark-orange text-light">Dark Orange</div>
+    <div class="p-3">&nbsp;</div>
+</div>
+
+<div class="mb-2 col-3" style="float:left;">
+    <div class="p-3 bg-green">Green</div>
+    <div class="p-3 bg-green-80">80%</div>
+    <div class="p-3 bg-green-60">60%</div>
+    <div class="p-3 bg-green-40">40%</div>
+</div>
+
+<div class="mb-2 col-3" style="float:left;">
+    <div class="p-3 bg-red">Red</div>
+    <div class="p-3 bg-red-80">80%</div>
+    <div class="p-3 bg-red-60">60%</div>
+    <div class="p-3 bg-red-40">40%</div>
+</div>
+
+<div class="mb-2 col-3" style="float:left;">
+    <div class="p-3 bg-yellow">Yellow</div>
+    <div class="p-3 bg-yellow-80">80%</div>
+    <div class="p-3 bg-yellow-60">60%</div>
+    <div class="p-3 bg-yellow-40">40%</div>
+</div>
+
+<div class="mb-2 col-3" style="float:left;">
+    <div class="p-3 bg-cyan">Cyan</div>
+    <div class="p-3 bg-cyan-80">80%</div>
+    <div class="p-3 bg-cyan-60">60%</div>
+    <div class="p-3 bg-cyan-40">40%</div>
+</div>
```

Comment:
new line at the end

Reply:
Thanks. Already changed now locally, but waiting, if there is more...